Loop/Extensions/NSUserDefaults.swift

```diff
@@ -10,15 +10,17 @@ import Foundation
 import LoopKit
 import MinimedKit
 import HealthKit
+import CarbKit
 
 extension UserDefaults {
-
+    
     private enum Key: String {
         case basalRateSchedule = "com.loudnate.Naterade.BasalRateSchedule"
         case batteryChemistry = "com.loopkit.Loop.BatteryChemistry"
         case cgmSettings = "com.loopkit.Loop.cgmSettings"
         case carbRatioSchedule = "com.loudnate.Naterade.CarbRatioSchedule"
         case connectedPeripheralIDs = "com.loudnate.Naterade.ConnectedPeripheralIDs"
+        case defaultAbsorptionTimes = "com.loudnate.Naterade.DefaultAbsorptionTimes"
```

Comment:
Could yout put this data into loopSettings? That will limit keys to make migration easier.